app/lib/features/space/settings/pages/notification_configuration_page.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:acter/common/widgets/with_sidebar.dart';
```

Comment:
Yeah. I was in the process of allowing for space-specific notification overwrites. That the user can overwrite the global settings, e.g. "No notifications for updates" with a local alternative "but Updates for this space". Unfortunately this wasn't that easily doable for the matrix side of things and I chose to push what we have now rather than figuring out that bit as well. However the refactor I had done around the widgets seem worth keeping, so I kept that code. I could move it back to other screen though we probably want to re-introduce this screen pretty soon either way ...

Reply:
hmm... as far as we can planning to do further things very soon here in this screen, no need to move it back. Let's keep this as it is.